### PR TITLE
Fix VFlow empty child wrapper rendered in DOM

### PR DESCRIPTION
**Pull request by gustavomoser**
Some empty `<div />` were added as a wrapper for each child in the old VFlow implementation even when child is empty. 

To fix this bug, I chose to remove the `React.Children.map`, letting the default type filtering act in children of VFlow, and don't rendering "invalid" children (like null, undefined, booleans).

Besides that, the VFlow became simpler, since it's no longer necessary to loop through the children just to add a `<div />` wrapper. Also, VFlow children started using all the horizontal space because of this wrapper removal.

### CODE REVIEW

File: src/components/VFlow/VFlow.test.tsx [hunk 1]
```diff
@@ -25,3 +25,18 @@ it('should accept "style" prop', () => {
...
+      {null}
+      {undefined}
+      {true}
+      {false}
+      {0}
+      {1}
+      {''}
+    </VFlow>
+  )
+  expect(container.querySelectorAll('div')[0].childNodes.length).toEqual(3)
```
Comment (lucasjoao) [inline src/components/VFlow/VFlow.test.tsx:41]: what do you think about adding a "expect line" to check the snapshot too?

**Review by lucasjoao — APPROVED** [review]
:rocket: 

Just one detail in the test file to see

File: src/components/VFlow/VFlow.test.tsx (hunk 1 above)
Comment (gustavomoser) [reply src/components/VFlow/VFlow.test.tsx:41]: It's a good idea, I'll do it :+1:

**Review by lucasjoao — APPROVED** [review]
:rocket:

**Pull request closed without merging**